primitives/Cargo.toml

```diff
@@ -4,22 +4,29 @@ version = "0.6.0"
 authors = ["Supercomputing Systems AG <info@scs.ch>"]
 license = "Apache-2.0"
 edition = "2021"
+repository = "https://github.com/scs/substrate-api-client"
+description = "Substrate-api-client primitive types"
+readme = "README.md"
+categories = ["no-std"]
+
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ['derive'] }
-hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
 impl-serde = { version = "0.4", default-features = false }
 primitive-types = { version = "0.12.1", default-features = false, features = ["serde_no_std", "scale-info"] }
 scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
 
 # substrate no_std
-sp-core = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master" }
+sp-core = { default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "master" }
```

Comment:
I assume these changes are needed to get `no_std` working?

Reply:
Yes, exactly. The trait `Pair` 
https://github.com/scs/substrate-api-client/blob/6d09a9de7ce0fe7848d1bf2c206259bb1eceaea3/primitives/src/extrinsics/signer.rs#L23

is only available in std or `full_crypto`. The reason it compiled without problems until now, is that the `full_crypto` feature has been added by the importing crates.